scripts/high_level_motion_unit_test.py

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env python
+import sys
+import rospy
+from iarc7_msgs.msg import OdometryArray
+import actionlib
+import tf2_ros
+from iarc7_motion.msg import QuadMoveGoal, QuadMoveAction
+from iarc7_safety.SafetyClient import SafetyClient
+
+def test():
+    safety_client = SafetyClient('high_level_motion_unit_test')
+    # Since this abstract is top level in the control chain there is no need to check
+    # for a safety state. We can also get away with not checking for a fatal state since
+    # all nodes below will shut down.
+    assert(safety_client.form_bond())
+
+    # Creates the SimpleActionClient, passing the type of the action
+    # (QuadMoveAction) to the constructor. (Look in the action folder)
+    client = actionlib.SimpleActionClient("motion_planner_server", QuadMoveAction)
+
+    # Waits until the action server has started up and started
+    # listening for goals.
+    client.wait_for_server()
+
+    rospy.sleep(2.0)
+
+    # change element in array to test diff roombas
+    roomba_id = roomba_array.data[5].child_frame_id 
+    roomba_id = roomba_id [0:len(roomba_id)-10]
```

Comment:
No space before the `[` please, also please comment why there's a `10` here

Reply:
Fixed